src/Lang/Surface.ml

```diff
@@ -301,6 +301,9 @@ and def_data =
     (** Print type, evaluate, and print the expression, provided by a user in
       REPL. *)
 
+  | DReplDefs of def list
+    (** Group of REPL definitions *)
```

Comment:
Why do you add a new constructor? It is easier to modify `ERepl` to accept list of definitions.